ArraysHomework/ArraysHomework.cs

```diff
@@ -10,11 +12,12 @@ public class ArraysHomework
         public void Arrays()
         {
             // Create an array of ints of size 100
-            
+            int[] myArray = new int[100];
+
             Assert.IsTrue(myArray.Length == 100);
 
             // take the array made above and fill it so these tests pass  
-            
+
```

Comment:
The values aren't being filled in (not all indices need a value assigned, but the ones that are being tested do).